examples/medplum-chart-demo/README.md

```diff
@@ -8,12 +8,25 @@
 
 This example app demonstrates the following:
 
-- Creating a new React app with Vite and TypeScript
-- Adding Medplum dependencies
-- Adding basic URL routing
-- Using the [Medplum client](https://www.medplum.com/docs/sdk/classes/MedplumClient) to search for FHIR resources
+- Using [Medplum React Components](https://storybook.medplum.com/?path=/docs/medplum-introduction--docs) to display a chart that provides visibility on a patient
```

Comment:
Should we add https://www.medplum.com/docs/charting to the README? I'm worried the image looks different than the chart in the examples